EKF/common.h

```diff
@@ -204,6 +209,7 @@ struct parameters {
 	// measurement source control
 	int32_t fusion_mode{MASK_USE_GPS};		///< bitmasked integer that selects which aiding sources will be used
 	int32_t vdist_sensor_type{VDIST_SENSOR_BARO};	///< selects the primary source for height data
+	int32_t terrain_fusion_mode{3};			///< aiding source(s) selection bitmask for the terrain estimator
```

Comment:
It doesn't really matter, but you could use the actual bits here.

```suggestion
	int32_t terrain_fusion_mode{TerrainFuseOpticalFlow | TerrainFuseRangeFinder};			///< aiding source(s) selection bitmask for the terrain estimator
```

Reply:
Yes, thanks